db/seeds.rb

```diff
@@ -134,6 +133,16 @@ def create_ht_registration(inst_id)
     mfa_addendum: [true, false].sample,
     contact_info: Faker::Internet.email
   )
+  if rand < 0.5
```

Comment:
I'd consider making these "sent" and "received" traits on the `ht_registration` factory and then doing something like

```ruby
[build(:ht_registration), build(:ht_registration,:sent), build(:ht_registration,:received)].sample.save!
```

Reply:
I think I'm going to disagree on this one based on what I understand about FactoryBot: we are probably creating factory objects that are too completely specified, and tests should manipulate factory objects that have only the fields that are `NOT NULL` or are germane to the test at hand. Currently our seeding/factorying practices are eerily similar but I think the push should not be to merge them.

Reply:
Fine by me.